src/games/strategy/triplea/settings/ValueRange.java

```diff
@@ -0,0 +1,23 @@
+package games.strategy.triplea.settings;
+
+/**
+ * Encapsulates the permissible range for the value of a specific setting.
+ *
+ * <p>
+ * Implementations may choose to track other associated values in addition to the range itself, such as a default value.
+ * </p>
+ */
+public interface ValueRange {
```

Comment:
Is this interface really worth it at this point? What is the value we get currently? Guess I'm wondering whether this is kind of over engineering it given that it has one method and is only implemented in one place.